src/common/hackathonBanner/HackathonBanner.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
```

Comment:
- change the folder name to `activities`
- Inside that create a folder called `hackathon`
- Inside that this file should be there.